site/src/sphinx/client-java.rst

```diff
@@ -515,17 +515,73 @@ client profile JAR again.
 Spring Boot integration
 -----------------------
 If you are using `Spring Framework <https://spring.io/>`_, you can inject :api:`com.linecorp.centraldogma.client.CentralDogma`
-client very easily.
+client very easily. First, add ``centraldogma-client-spring-boot-autoconfigure`` into your dependencies.
```

Comment:
nit: How about using `centraldogma-client-spring-boot-starter`? They are same but we have `-starter` module too, so I think it would be better to use `-starter` module.

Reply:
You're right. Fixed.